README.md

```diff
@@ -31,16 +31,20 @@ Or install it yourself as:
 
     $ gem install npmdc
 
+To use npmdc with yarn you should install yarn.
+
 ## Usage
 
 ### Rails
 
 ```ruby
 YourApp::Application.configure do
+  config.npmdc.package_manager  = "yarn"                            # `npm` by default
+  config.npmdc.path_to_yarn     = "/path/to/your/yarn"              # nil by default, so the command is just `yarn`
   config.npmdc.path             = "/path/to/your/frontend/code/dir" # `Rails.root` by default
   config.npmdc.format           = "doc"                             # `short`, `doc`, `progress`. `short` by default
   config.npmdc.color            = false                             # `true` by default
-  config.npmdc.types            = ["dependencies"]                  # `["dependencies", "devDependencies"]` by default
+  config.npmdc.types            = ["dependencies"]                  # `["dependencies", "devDependencies"]` by default (ignored with yarn package_manager)
```

Comment:
@sponomarev it seems entire `npmdc.types` option, because of `yarn check` implementation there is no any built in method to check dependences & devDependencies separately. 

Also I have no idea why we've provided ability to check dependencies & devDependencies separately. There is no real life use case for that 😀 Let's leave this "as is" for now.